mod-pubsub-client/pom.xml

```diff
@@ -198,7 +183,10 @@
             <phase>initialize</phase>
             <configuration>
               <sources>
-                <source>${project.build.directory}/generated-sources</source>
+                <!-- Adds client classes generated in current client module-->
+                <source>${project.build.directory}/generated-sources/raml-jaxrs</source>
```

Comment:
clients not generating here, so we dont need to include them

Reply:
The line is moved as well, thanks